README.md

```diff
@@ -1,150 +1,232 @@
 
-If you wish to work on Core, simply run `npm run start`.
+To set up the origin, use the following command in the Terminal:
 
-2.  Netlify Proxy + Core :
+    git remote add origin [origin URL]
 
-    1.  To work on cms configs, run:
+Replace `[origin URL]` with the git repository URL of your repository. Now, after setting up the origin, you need to add the `upstream`. To do that, use the following command:
 
-        ```sh
-        npm run start:auth
-        ```
+    git remote add upstream [upstream URL]
 
-    2.  In a separate terminal, this will start the netlify-cms-proxy-server and will enable you to work with the cms locally.
-        and then run:
+Replace `[upstream URL]` with the git repository URL for the main [repository](https://github.com/deriv-com/deriv-api-docs.git).
 
-        ```sh
-        npm run start
-        ```
+### Branching and Pull Request Procedure
 
-<br />
+To create a branch from the latest master branch, follow these steps:
 
-## How to Contribute
+1. Switch to the master branch using the command `git checkout master`.
+2. Pull the latest changes from the upstream master branch using the command `git pull upstream master`.
+3. Create a new branch with your desired branch name using the command `git checkout -b [your_branch_name]`.
+4. Push your changes to the origin branch using the command `git push origin [your_branch_name]`.
+
+### Preview Pull Request - Testing/QA Verification
+
+Upon creating a PR, [Vercel](https://vercel.com/) will auto-generate a test link inside the PR. You can use that to preview the test link for the changes you have made.
+
+## Packages Used
+
+In addition to `Node`, `npm`, and `git`, we have utilized various external and internal packages in this project.
+
+#### Docusaurus
+
+Docusaurus is a modern static site generator designed for creating documentation websites. It simplifies the process of building, deploying, and maintaining documentation projects. With Docusaurus, you can organize your documentation, customize the layout and styling, and collaborate seamlessly with version control systems. It offers search functionality, responsive design, and support for multiple languages. Docusaurus is widely used by developers and organizations to create professional and user-friendly documentation websites. Give it a try and experience its power for your documentation needs. For more information, refer to the official [documentation](https://docusaurus.io/docs/3.3.2).
+
+#### @Deriv/ui
+
+The `@deriv/ui` package provides a collection of reusable UI components for building web applications.
+It includes a wide range of components such as buttons, forms, modals, and more, that can be easily customized and integrated into your project.
+
+This package follows a modular approach, allowing you to import and use only the components you need, reducing the bundle size and improving performance.
+
+To install the `@deriv/ui` package, you can use npm or yarn:
+
+```shell
+npm install @deriv/ui
```

Comment:
@ahmed-deriv We don't use deriv ui anymore, it's all being migrated to @deriv-com/quill-ui in the feature branch. So we can can include that here itself.

Please correct me if I'm wrong @sandeep-deriv @shafin-deriv

Reply:
The reason I added here is that currently in master branch we are not using @deriv-com/quill-ui. So I assume that we can update that later. otherwise it might create misunderstanding.

Reply:
Sure, makes sense.